storage/series.go

```diff
@@ -239,9 +265,13 @@ func (s *seriesToChunkEncoder) Iterator() chunks.Iterator {
 	maxt := int64(math.MinInt64)
 
 	chks := []chunks.Meta{}
```

Comment:
Does this cause an allocation of a slice? Should this be `var chks []chunks.Meta` now?